src/components/EmptyState/EmptyStatePattern.test.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-env jest */
+
+import React from 'react';
+import ShallowRenderer from 'react-test-renderer/shallow';
```

Comment:
can we use enzyme shallow() now?

Reply:
also, yah shouldn't need `/* eslint-env jest */`